tests/message/message_renames.proto

```diff
@@ -7,30 +7,44 @@ import "patch/go.proto";
 option go_package = "github.com/alta/protopatch/tests/message";
 
 message Francis {
-	option (go.message).name = 'Frank';
+  option (go.message).name = 'Frank';
```

Comment:
Please format these consistently with the existing files. I use the clang-format extension in VS Code to automatically format proto files according to the spec in `.clang-format`.